xkcd_downloader.py

```diff
@@ -0,0 +1,46 @@
+"""
+Written by: Shreyas Daniel - github.com/shreydan
+Written on: 26 April 2017
+
+Description: Download latest XKCD Comic with this program.
+
+NOTE:
+	if this script is launched from the cloned repo, a new folder is created.
+	Please move the file to another directory to avoid messing with the folder structure.
+"""
+
+import requests
+from lxml import html
+import urllib.request
+import os
+
+# opens xkcd.com
+try:
```

Comment:
Good to organise the code below inside a function or inside __name__ == '__main__'